include/antidote_locks.hrl

```diff
@@ -0,0 +1,58 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright <2013-2018> <
```

Comment:
Shouldn't this become 2019?

Reply:
It's the same in other files. I can create a separate PR to update the date everywhere.